docker/validator-dynamic/docker-run-dynamic.sh

```diff
@@ -42,7 +42,7 @@ fi
     ${params[@]}
 
 
-if [ -n "${CFG_FULLNODE_SEED}" ]; then # We have a full node seed, add fullnode network
+if [ -n "${CFG_FULLNODE_SEED}" -a "${CFG_ENABLE_MGMT_TOOL}" = false ]; then # We have a full node seed, add fullnode network
```

Comment:
was this tested? cc: @snakech

Reply:
I was not there when this was merged, but FWIW this file is no longer used by cluster test, so if it breaks something, feel free to just revert the change

Reply:
I confirmed with @changyonghu he did the test. And since we do not use config builder in CT anymore, I guess its ok to remove this flag

Reply:
@davidiw  CT uses this file before, since @andll already removed config_builder in CT, it's ok to remove this flag.

Reply:
I used this flag to ensure both new version (with mgmt tool) and old version works.

Reply:
> @davidiw CT uses this file before, since @andll already removed config_builder in CT, it's ok to remove this flag.

gosh, how do you pop up here? for after sale service I assume aha

Reply:
Thanks for following up. The problem with this check is that it breaks compatibility with tools that are unaware of the change. The safer approach would be to just check if it is set or not. Now if setting it is mandatory, the alternative would be = false or not set. See https://github.com/libra/libra/pull/5793/files

Reply:
Good to know. Not very familiar with bash. I thought it was set to false in default.

Reply:
Bash is a very painful experience... :D